src/modules/command/modifier.rs

```diff
@@ -40,11 +40,17 @@ impl CommandModifier {
             match meta.get_current_token() {
                 Some(tok) => {
                     match tok.word.as_str() {
-                        "unsafe" => {
-                            if self.is_unsafe {
-                                return error!(meta, Some(tok.clone()), "You already declared `unsafe` modifier before");
+                        trust @ ("trust" | "unsafe") => {
+                            if trust == "unsafe" {
+                                let message = Message::new_warn_at_token(meta, Some(tok.clone()))
+                                .message("The keyword `unsafe` has been deprecated in favor of `trust`.")
```

Comment:
Out of interest, and not asking you to change anything... but do we have any guidelines on whether to terminate messages with periods or not?

Reply:
I don't think that we have any guidelines for messages.
It's stuff for a ticket or discussion.